vane-regions/src/main/java/org/oddlama/vane/regions/menu/RoleMenu.java

```diff
@@ -192,7 +195,10 @@ public Menu create(final RegionGroup group, final Role role, final Player player
 		add_menu_item_setting(role_menu, role, 2, item_setting_info_build, RoleSetting.BUILD);
 		add_menu_item_setting(role_menu, role, 4, item_setting_info_use, RoleSetting.USE);
 		add_menu_item_setting(role_menu, role, 5, item_setting_info_container, RoleSetting.CONTAINER);
-		add_menu_item_setting(role_menu, role, 8, item_setting_info_portal, RoleSetting.PORTAL);
+
+		if (item_setting_info_portal != null) {
```

Comment:
Instead of `item_setting_info_portal != null` you can use `get_module().region_portal_integration != null`

Reply:
Maybe there are other menus where the same logic applies.